src/com/company/medicine/Checkup.java

```diff
@@ -0,0 +1,73 @@
+package com.company.medicine;
+
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Checkup extends MedicalProcedure {
+    protected Affection diagnosis;
+    protected List<Treatment> treatments = new ArrayList<>();
+    protected String motive;
+    protected String observations;
+
+    public Checkup(LocalTime start_time, String motive) {
```

Comment:
fara _ in nume de parametri, iar in nume de campuri doar daca sunt contante (static, final) si alea se scriu cu litere mari